src/fruity/dtx.vala

```diff
@@ -62,6 +62,10 @@ namespace Frida.Fruity {
 				if (process.get_optional_value ("startDate", out start_date))
 					info.start_date = start_date.to_date_time ();
 
+				NSNumber? is_application;
+				if (process.get_optional_value ("isApplication", out is_application))
+					info.is_application = is_application.boolean;
```

Comment:
- Let's use `get_value()` here as we can assume it's already there.
- Perhaps we should move it before `foreground_running` as I guess that is a property that only applies to apps. (Let's also match the order here with that of the properties.)